src/lib/commands/config.ts

```diff
@@ -18,18 +19,24 @@ const showRenameAccountPrompt = (accountToRename: Portal) => {
     .showInputBox({
       placeHolder: 'Enter a new name for the account',
     })
-    .then((newName: string | undefined) => {
+    .then(async (newName: string | undefined) => {
```

Comment:
nit: could this just be optional?

Reply:
I'm not sure that I follow. When we are renaming an account, we need the `newName`, right?

Reply:
sorry I just mean could `newName: string | undefined` be `newName?: string`

Reply:
Thanks for clarifying, that makes total sense. I don't think it really matters, though I do think that `string | undefined` is a bit more explicit and lends itself to later updating to something like `InputBoxResponseType` or something like that. 🤷